content/kubermatic/main/tutorials-howtos/kubeone-clusters/_index.en.md

```diff
@@ -0,0 +1,105 @@
++++
+title = "KKP KubeOne Integration"
+date = 2023-02-21T14:07:15+02:00
+weight = 7
+
++++
+
+This section describes how to import and manage KubeOne clusters in KKP.
+We can import/connect an existing KubeOne cluster. Imported Cluster can be viewed and edited.
+
+  Currently Supported Providers:
+   - [AWS]({{< ref "./aws" >}})
+   - [Google Cloud Provider]({{< ref "./gcp" >}})
+   - [Azure]({{< ref "./azure" >}})
+
+## Prerequisites
+
+The following requirements must be met to import a KubeOne cluster:
+ - The KubeOne cluster must already exist before we begin the import/connect process.
+ - Manifest: KubeOne can define our KubeOne clusters declaratively, in a form of a YAML manifest.
+   Run `kubeone config dump -m kubeone.yaml -t tf.json` in our KubeOne terraform directory to get a manifest.
+ - Private SSH Key used to create the KubeOne cluster:  KubeOne connects to instances over SSH to perform any management   operation.
+ - Provider Specific Credentials used to create the cluster.
+
```

Comment:
if this needs special attention, I would use `{{% notice info %}}` rather than citation (`>`)